docs/policygenerator-reference.yaml

```
@@ -21,6 +21,10 @@ policyDefaults:
   # annotation. This defaults to ["CM-2 Baseline Configuration"].
   controls:
     - "CM-2 Baseline Configuration"
+  # Optional. Default value true will consolidate multi manifests 
+  # specified in policyConf into single policy template. 
+  # Set to false will skip this consolidation.
+  consolidatedManifests: true
```

Could you make the option name an action/verb to match the rest of the options? I'm sorry I didn't catch this previously.
```suggestion
  consolidateManifests: true
```